ci/deploy-forklift-plugin.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+script_dir=$(dirname "$0")
+
+K8S_TIMEOUT=${K8S_TIMEOUT:="360s"}
+
+FORKLIFT_PLUGIN_CONSOLE_YAML=${script_dir}/yaml/forklift-console-plugin.yaml
+FORKLIFT_PLUGIN_UPSTREAM_IMG=quay.io/kubev2v/forklift-console-plugin:latest
+FORKLIFT_PLUGIN_LOCAL_IMG=localhost:5001/forklift-console-plugin:latest
+
+# Install OKD console
+# -------------------
+echo ""
+echo "Starting OKD console"
+echo "===================="
+
+echo ""
+echo "deploy forklift console plugin CRDs"
+
+kubectl apply -f ${script_dir}/yaml/crds/forklift
+
+echo ""
+echo "deploy forklift console plugin"
+
+cat ${FORKLIFT_PLUGIN_CONSOLE_YAML} | \
```

Comment:
@rszwajko
hmm, but I want it to be usable ...
does it make sense to to have template AND a regular yaml ?

Reply:
AFAIK there are 2 choices here: templating  or overlays. For templating we need a template which alone is not a valid yaml. In case of overlays (using https://kustomize.io/) the base is a valid yaml. Perhaps this is our way?

Reply:
:+1: good excuse for learning https://kustomize.io/ :-)